src/commands/NodeActionCommands.tsx

```diff
@@ -910,4 +910,93 @@ export function addNodeActionCommands(
         
         }
     }
+
+    // Add command to attach selected node
+    commands.addCommand(commandIDs.attachNode, {
+        execute: async () => {
```

Comment:
I think this command should have a guard to not try and attach any nodes that aren't literals.

Reply:
Updated so only entities that starts with "Literal  " is considered.